src/main/java/de/tum/in/www1/artemis/service/CourseService.java

```diff
@@ -128,11 +149,12 @@ public Course findOneWithExercisesAndLecturesForUser(Long courseId, User user) {
      * @return the list of all courses including exercises and lectures for the user
      */
     public List<Course> findAllActiveWithExercisesAndLecturesForUser(User user) {
-        return courseRepository.findAllActiveWithLecturesAndExams().stream()
+        return courseRepository.findAllActiveWithLecturesAndLectureUnitsAndExams().stream()
```

Comment:
Do we really neueste load the exam here? The method names seem inconsistent to me

Reply:
Done in a76cea4f274c6a1fba1c6b1ceb24c7e449c4089d